tools/downloader/topologies.py

```diff
@@ -0,0 +1,81 @@
+import os
+import argparse
+import sys
+
+from pathlib import Path
+from argparse import Namespace
+
+import common
+import downloader
+import converter
+
+_cache = Path(os.getenv('OPEN_MODEL_ZOO_CACHE_DIR', Path('.').resolve() / 'models'))
+_cache_dir = _cache / 'models_cache'
+
+def load_model(model_name, precision='FP32'):
```

Comment:
Also, I think `load_model` would be great as `Topology` method.

Reply:
Can you show example how user's script may looks like in this way?

Reply:
```
from topologies import Topology
...
ssd = Topology(name, path_to_model)
ssd.load() 
model = ssd.getIR(precision)
```
Or `load` could be inside constructor.

Reply:
The thing is that constructor is already used to define the topology (there are a lot of arguments such doc string etc.)

Reply:
@likholat, we just discusses with @eizamaliev that now we have `load_model` method which directly calls `Topology` constructor. So it's a kind of redundant chain and more simple to have
```
ssd = omz.Topology('mobilenet-ssd')
```
which invocated downloader inside.